docs/libcurl/opts/CURLOPT_NOSIGNAL.3
=======================================================
lhs: 100644 | 21969e359eccb35f26481835a75983f08eefe979
rhs: 100644 | 27fe1580287b20677fc1aa539ddee534fd038995
---@@ -31,8 +31,7 @@ CURLcode curl_easy_setopt(CURL *handle, CURLOPT_NOSIGNAL, long onoff);
 If \fIonoff\fP is 1, libcurl will not use any functions that install signal
 handlers or any functions that cause signals to be sent to the process. This
 option is here to allow multi-threaded unix applications to still set/use all
-timeout options etc, without risking getting signals.  The default value for
-this parameter is 0.
+timeout options etc, without risking getting signals.
 
 If this option is set and libcurl has been built with the standard name
 resolver, timeouts will not occur while the name resolve takes place.
@@ -48,5 +47,9 @@ have no way to avoid them and even on those that have there are some corner
 cases when they may still happen, contrary to our desire. In addition, using
 \fICURLAUTH_NTLM_WB\fP authentication could cause a SIGCHLD signal to be
 raised.
+.SH DEFAULT
+0
 .SH AVAILABILITY
 Added in 7.10
+.SH RETURN VALUE
+Returns CURLE_OK if the option is supported, and CURLE_UNKNOWN_OPTION if not.

---